splunk_topology/check.py

```diff
@@ -197,7 +214,12 @@ def _dispatch_saved_search(self, instance, saved_search):
 
         self.log.debug("Dispatching saved search: %s." % saved_search.name)
 
-        return instance.splunkHelper.dispatch(saved_search, splunk_user, splunk_app, splunk_ignore_saved_search_errors, parameters)
+        sid = instance.splunkHelper.dispatch(saved_search, splunk_user, splunk_app, splunk_ignore_saved_search_errors, parameters)
+        if self.status.data.get(instance.instance_config.base_url) is None:
+            self.status.data[instance.instance_config.base_url] = []
+        self.status.data.get(instance.instance_config.base_url).append((sid, saved_search))
```

Comment:
can this reuse the extracted method `update_status` in the splunk_telemetry_base.py ?